go-controller/pkg/config/config.go

```diff
@@ -405,7 +405,7 @@ func buildKubernetesConfig(cli, file *config, defaults *Defaults) error {
 		return fmt.Errorf("kubernetes kubeconfig file %q not found", Kubernetes.Kubeconfig)
 	}
 	if Kubernetes.CACert != "" && !pathExists(Kubernetes.CACert) {
-		return fmt.Errorf("kubernetes CA certificate file %q not found", Kubernetes.CACert)
+		Kubernetes.CACert = ""
```

Comment:
If someone has specified it in CLI, would it not make sense for it to exist? is there  a usecase where this is useful?

Reply:
CACert could also be retrieved from ovsdb external_ids. I have a use case where 

"k8s-ca-certificate"="/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"

is injected there because of init-node running in container as a daemonset. Let me remove this change and find more elegant solution.